test/Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests/LanguageEndToEndTestExtensions.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests
+{
+    internal static class LanguageEndToEndTestExtensions
+    {
+        public static KafkaEventData<string> KafkaEventData(this string s)
```

Comment:
Perhaps rename it to 'ToKafkaEventData' or 'AsKafkaEventData', as it might make it clearer that it converts a string to a KafkaEventData.

Reply:
Done